File: app/src/androidTest/java/com/przemolab/oknotifier/modules/TestOpenKattisServiceModule.java
package com.przemolab.oknotifier.modules;

import org.mockito.Mockito;

public class TestOpenKattisServiceModule extends OpenKattisServiceModule {

    @Override
    public OpenKattisService provideOpenKattisService() {
        return Mockito.mock(OpenKattisService.class);
    }
}

File: app/src/androidTest/java/com/przemolab/oknotifier/modules/TestOpenKattisServiceModule.kt
package com.przemolab.oknotifier.modules

import com.przemolab.oknotifier.interfaces.IOpenKattisService
import org.mockito.Mockito

class TestOpenKattisServiceModule : OpenKattisServiceModule() {

    override fun provideOpenKattisService(): IOpenKattisService {
        return Mockito.mock(IOpenKattisService::class.java)
    }
}
